locale/en/blog/npm/peer-dependencies.md

```diff
@@ -124,6 +124,10 @@ peer-depend on, and add it to your `package.json`:
 Now, when installing `chai-as-promised`, the `chai` package will come along with it. And if later you try to install
 another Chai plugin that only works with 0.x versions of Chai, you'll get an error. Nice!
 
+<div class="highlight-box">
+  <strong>UPDATE:</strong> npm versions 1 and 2 will automatically install peerDependencies if they are not explicitly depended upon higher in the dependency tree. For all following versions of npm (starting with npm@3), this will no longer be the case. You will receive a warning that the peerDependency is not installed instead.
```

Comment:
small nit `depended` --> `dependant`

Reply:
@daxlab Thanks, fixed!

Reply:
@daxlab Are you sure this was typo? I think "depended" was correct.

Reply:
@lpinca Yes, you're right … :confused: